### ISSUE TITLE
Principe du code

### ISSUE THREAD

**Issue by Vayel** [Vayel/devin#2]
* Lire le fichier .ods des Chaffs avec Python
* Générer la page HTML du devis à partir d'un template (Jinja 2 ?)

**Comment by Vayel** [comment]
@XAMEUS as-tu un avis sur la question ? Python est simple d'utilisation et très portable. De plus, on aura accès à toutes les bibliothèques qu'on souhaite pour nous simplifier le travail.

**Comment by XAMEUS** [comment]
@Vayel Oui, Python semble très bien (surtout que tu gères super bien, non ?).
Après il était question de développer un CRM pour la junior ESC ; ce même CRM pourrait ensuite être utilisé par Nsigma. À voir si ce module pourrait s'intégrer dans quelque chose de plus gros.
